lib/round.rb

```diff
@@ -0,0 +1,42 @@
+class Round
+  attr_reader :deck,
+              :turns
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+  end
+
+  def current_card
+    @deck.cards[turns.count]
```

Comment:
I like how you use the number of turns to keep track of what the current card should be, very smart.